backend/src/routes/songs.js

```diff
@@ -10,19 +10,22 @@ async function handleSong(req, res) {
     return res.status(400).send('Insufficient info');
   }
   // TODO: Add data validation
-  const result = await db.collection('songEntries').add({
+  const songDoc = {
     songData,
     locationHash: geofire.geohashForLocation([lat, lng]),
     lat,
     lng,
     ts: firebase.firestore.FieldValue.serverTimestamp(),
-  });
+  };
+  const result = await db.collection('songEntries').add(songDoc);
```

Comment:
We may need to dedupe songEntries in your area if many people in the area submit the same song. Is it possible to filter unique on a field of a field (like get all unique songData.songId within a radius)

Reply:
@sjiang647 do you think we can add deduping logic in the /nearby endpoint to only get unique songEntries